src/components/Quotes/Quotes.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+
+function Quotes() {
+  const [quote, setQuote] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    const URL = 'https://api.quotable.io/random';
+    setIsLoading(true);
+    setHasError(false);
+
+    fetch(URL)
+      .then((response) => response.json())
+      .then((data) => {
+        setQuote(data.content);
+        setIsLoading(false);
+      })
+      .catch(() => {
+        setHasError(true);
+        setIsLoading(false);
+      });
```

Comment:
#### [OPTIONAL]
You can also use `async await` syntax to enhance the readability of your code. 👍